ai-platform-unified/notebooks/ucaip_customjob_image.ipynb

```diff
@@ -0,0 +1,2540 @@
+{
```

Comment:
<p>Nit: In the cell below, we have</p><p> </p><p><code></code><code></p><p>if precook:</p><p>&nbsp;&nbsp;model_path_to_deploy = "[not-yet-implemented]"</p><p></code><code></code></p><p> </p><p>Since it seems like the "pre-cooked" option is not currently usable, can we remove mention of it? (Also, you could remove the pre-cooked option from the code cell.) Example for the text:</p><p> </p><pre class="ql-syntax" spellcheck="false">## Deployment

Once your model is done training, you can calculate the actual time it took to train the model by subtracting start_time from end_time. For your model, we need to know the location of the SavedModel, which the Python script saved in your Cloud Storage bucket at MODEL_DIR + '/saved_model.pb'.
</pre><p>(I swapped <code>start_time</code> and <code>end_time</code> to make more sense and did a little other cleanup 😊)</p><p> </p><p>If you plan to add the pre-cooked option soon, feel free to ignore this comment; I just think it's nicer not to offer customers an option that they can't use.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/240/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='78'/>